snakemake_executor_plugin_slurm/accounts.py

```
@@ -0,0 +1,78 @@
+            f"accounts:\n{', '.join(accounts)}"
+        )
+
+
+def get_account(logger):
+    """
+    tries to deduce the acccount from recent jobs,
+    returns None, if none is found
+    """
+    cmd = f'sacct -nu "{os.environ["USER"]}" -o Account%256 | tail -1'
+    try:
+        sacct_out = subprocess.check_output(
```

Ruff (0.15.1)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)